File: src/lightvirus/baekjoon/_4344.java
package lightvirus.baekjoon;

import java.util.Scanner;

public class _4344 {
    public static void main (String [] args) {
        Scanner sc = new Scanner(System.in);

        int C = sc.nextInt(); // 테스트 케이스 개수
        int N = 0; // 학생의 수

        for (int i = 0; i < C; i ++) {
            N = sc.nextInt();
            int score[] = new int[N];
            int sum = 0;
            for (int j = 0; j < N; j ++) {
                score[j] = sc.nextInt();
                sum += score[j];
            }
            float avg = (float) sum / (float) N;
            int cnt = 0;
            for (int z = 0; z < N; z ++) {
                if (score[z] > avg) cnt++;
            }
            System.out.println(String.format("%.3f",((float)cnt / N*100)) + "%");
        }
    }
}

File: src/joyowlsf/baekjoon/_4344.py
a = int(input())

for i in range(a):
    num = list(map(int, input().split()))
    avg = sum(num[1:])/num[0]
    cnt = 0
    for i in range(1, len(num)):
        if num[i] > avg:
            cnt += 1

    print('%.3f' % (cnt / num[0] * 100) + '%')